sdk/cosmos/kafka.yml

```diff
@@ -7,7 +7,6 @@ extends:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
           TestName: 'Kafka_TestContainer_Integration'
-          UseFederatedAuth: false
           EnvVars:
             ACCOUNT_TENANT_ID: '72f988bf-86f1-41af-91ab-2d7cd011db47'
             ACCOUNT_AAD_CLIENT_ID: $(spark-databricks-cosmos-spn-clientId)
```

Comment:
@nehrao1 @kushagraThapar you should please work to remove this client id and client secret.

Reply:
Ack.

Reply:
We use this for our kafka connector live tests. I think this should be separated as our kafka connector does not support managed identity yet.

Reply:
yea it is used in java - cosmos - kafka